slurm/install/installlib.py

```diff
@@ -20,6 +29,42 @@ class ConvergeRetry(RuntimeError):
     pass
 
 
+class Clock:
```

Comment:
Same question here, is there some functionality we are trying to create here in this class?

Reply:
It is used so we can inject a mock clock for testing. See installlib_test.py